src/commands/export.ts

```diff
@@ -0,0 +1,160 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Alessandro Fragnani. All rights reserved.
+*  Licensed under the GPLv3 License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { commands, l10n, Uri, window, workspace } from "vscode";
+import { Container } from "../core/container";
+import { Controller } from "../core/controller";
+import { getRelativePath } from "../utils/fs";
+
+interface BookmarkExportItem {
+    file: string;
+    line: number;
+    column: number;
+    label: string;
+    content: string;
+}
+
+async function getLineContent(fileUri: Uri, lineNumber: number): Promise<string> {
```

Comment:
With the removal of `collectBookmark` function, this `getLineContent` should also be removed, since already exists a `getLinePreview` function that does exactly what is needed.